pkg/webui/console/components/events/messages.js

```diff
@@ -37,6 +37,9 @@ const messages = defineMessages({
   errorOverviewEntry:
     'There was an error and the event cannot be displayed. The raw event can by viewed by clicking this row.',
   dataPreview: 'Data preview',
+  dataFormats: 'Data Formats',
+  dataFormatsInformation:
+    'For more information on specific event message types, please see our {dataFormatsDocumentationLink} documentation.',
```

Comment:
```suggestion
    'For more information on event message types, please see our {dataFormatsDocumentationLink} documentation.',
```

Let's alter this a little to get rid of the line-break. If this still doesn't get rid of it, please try to reword further.